lib/createlocaltrack.js

```diff
@@ -58,7 +58,10 @@ function createLocalAudioTrack(options) {
 }
 
 /**
- * Request a {@link LocalVideoTrack}.
+ * Request a {@link LocalVideoTrack}. Note that on mobile browsers,
+ * the camera can be reserved by only one {@link LocalVideoTrack} at any given
+ * time. If you attempt to create a second {@link LocalVideoTrack}, video frames
+ * will no longer be supplied to the first {@link LocalVideoTrack}.
```

Comment:
Please add this blurb to the `createLocalTracks` description.

Reply:
you mean in addition to  `createLocalVideoTrack`  right, yep will do.